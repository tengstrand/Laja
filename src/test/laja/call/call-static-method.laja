#write "target/test-output/call-static-method.out"
  #set (map = {})
  #loadClass("net.sf.laja.integrationtest.CallTest$MyClass") into MyClass
  #MyClass.staticMethod(map, "x")
  #MyClass.staticMethod(22) into y
map.x = {map.x}
y = {y}
#end
