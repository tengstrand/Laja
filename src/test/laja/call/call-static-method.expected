map.x = Populated in MyClass.staticMethod(Map,String)
y = 122
